tools/cldr-code/src/test/java/org/unicode/cldr/util/TestCoverageLevel2.java

```diff
@@ -107,4 +127,71 @@ public void TestPriorBasicLanguage() throws IOException {
                                                 + " for these language's names:"
                                                 + notInCoverage.toString()));
     }
+
+    private void checkLanguageCoverageForScriptsInBasic(
+            final CalculatedCoverageLevels prevCovLevel,
+            final Map<String, CoverageLevel2> covs,
+            final CLDRFile english)
+            throws MultipleFailuresError {
+        final Level failIfAbove = Level.MODERN;
+
+        final String XPATH_LANGUAGE = "//ldml/localeDisplayNames/scripts/script";
```

Comment:
Odd name given that it is for scripts... and lines below refer to language when they really mean script. I know there was some copy-paste from the previous function...